src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,94 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+class RegistrationServiceTest {
+    private static RegistrationService registrationService;
+    private static User user;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        user = new User();
+    }
+
+    @BeforeEach
+    void setUp() {
+        Storage.people.clear();
+        user.setLogin("goodLogin");
+        user.setPassword("goodPassword");
+        user.setAge(18);
+    }
+
+    @Test
+    void validPassword_Ok() {
```

Comment:
format of test names should be different
[unitOfWork_stateUnderTest_ExpectedBehavior] - like this. 
for example: "registration_validPassword_Ok", instead of "validPassword_Ok"
raname all tests like in that

Reply:
Got ya